infer_structcol/tests/test_model.py

```diff
@@ -9,11 +9,11 @@
 from pandas.util.testing import assert_frame_equal
 
 def test_calc_model_spect():
-    sample = Sample(500, 100, 200, 1.5, 1)
-    theta = (0.5, 0, 0, 0, 0)
+    sample = Sample(500, 1.5, 1)
+    theta = (0.5, 100, 200, 0, 0, 0, 0)
     assert_frame_equal(calc_model_spect(sample, theta, 2), Spectrum(500, 
-                       reflectance = 0.813006364656, sigma_r = 0.025358376581, 
```

Comment:
I checked the value for 300 trajectories and the previous sigma, and the result is the same as it was before. So it is indeed due to the change in number of trajectories.